src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-5.2.ts

```diff
@@ -0,0 +1,105 @@
+//similar to CIS 6.3
+export default {
+  id: 'azure-nist-800-53-rev4-5.2',  
+  title: 'Azure NIST 5.2 PostgreSQL Database server firewall rules should not permit start and end IP addresses to be 0.0.0.0',
+  
+  description: 'Ensure that no SQL Databases allow ingress from 0.0.0.0/0 (ANY IP).',
+  
+  audit: `** From Azure Console**
+  
+  1. Go to SQL servers
+  2. For each SQL server
```

Comment:
I see that the documentation is the same as for SQL Server (5.4). It would have to be updated for PostreSQL. That's all.